README.md

```diff
@@ -10,6 +10,18 @@ This project provides tools for working with JSON values using [circe][circe] an
 it was promoted to its own project in order to decouple its releases (and the Scala versions it
 is cross-published for) from the main circe modules.
 
+## Versioning
+
+This library releases at a different cadence than [Circe], any apparent relationship between the [Circe] version numbers
+and circe-optics version numbers are a coincidence and NOT intentional.  
+Please use the following table for determining capability with the corresponding [Circe] versions.
+
+| Circe Optics Version | Circe  | Monocle | Scala 2 | Scala 3 | Scala JVM | Scala.JS |
+|----------------------|--------|---------|---------|---------|-----------|----------|
+| 0.15.0 and later     | 0.14.x | 3.x     | ✅      | ✅      | ✅        | ✅       |
+| 0.14.1               | 0.14.x | 2.x     | ✅      | ❌      | ✅        | ✅       |
+| 0.14.0               | 0.14.x | 2.x     | ✅      | ❌      | ✅        | ✅       |
+
```

Comment:
I don't want to hold up the PR for this change, but we need to split Scala 2 into 2.12, and 2.13 to make more clear that 0.15.0 does not support scala 2.12.

Reply:
Addressed by https://github.com/circe/circe-optics/commit/c0a6512cf11a9f28030e3fac2bca4d184ea5ee7e